spec/requests/posts_controller_spec.rb

```diff
@@ -1,39 +1,39 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Posts', type: :request do
-  describe 'get #index' do
-    it 'it is success' do
-      get user_posts_path(user_id: 1)
-      expect(response).to have_http_status(200)
-    end
+# RSpec.describe 'Posts', type: :request do
+#   describe 'get #index' do
+#     it 'it is success' do
+#       get user_posts_path(user_id: 1)
+#       expect(response).to have_http_status(200)
+#     end
```

Comment:
- [ ] [OPTIONAL] This is optional because it was not picked up by the previous reviewer. Kindly remove all commented lines of code as it makes your code look untidy 👍